x/epoching/spec/02_current_implementation.md

```diff
@@ -0,0 +1,35 @@
+<!--
+order: 2
+-->
+
+# Current Implementation
+
+## Messages queue
+
+Messages are queued to run at the end of epochs.
+Queued messages have epoch number to be run and at the end of epochs, it run messages queued for the epoch and execute the message.
```

Comment:
Which messages have things execute immediately, and what are these actions?

Reply:
Thanks for adding explanations for these!

Can you also add some text regarding if there one message queue, or a different queue per message type?